Sources/ActionSheet/ActionSheetCell.swift

```diff
@@ -9,6 +9,58 @@
  * Refer to the COPYING file of the official project for license.
  *****************************************************************************/
 
+enum ActionSheetCellAccessoryType: Equatable {
+    case none
+    case checkmark
+    case disclosureChevron
+    case custom(image: UIImage)
+    
+    static func == (lhs: ActionSheetCellAccessoryType, rhs: ActionSheetCellAccessoryType) -> Bool {
+        switch lhs {
+        case .checkmark:
+            switch rhs {
+            case .checkmark:
+                return true
+            default:
+                return false
+            }
+        case .disclosureChevron:
+            switch rhs {
+            case .disclosureChevron:
+                return true
+            default:
+                return false
+            }
+        case .custom( _):
+            switch rhs {
+            case .custom( _):
+                return true
+            default:
+                return false
+            }
+        case .none:
+            switch rhs {
+            case .none:
+                return true
+            default:
+                return false
+            }
+        default:
+            assertionFailure("Unhandled ActionSheetAccessoryType")
+        }
```

Comment:
I'm pretty sure there is a better way than this! Can you spend some more time researching? 
Additionally we like to avoid default to explicitly handle defined cases

Reply:
Can't this be removed? I'm not sure this overload is needed in this case.